src/buttons/_base.scss

```diff
@@ -159,38 +159,34 @@
 
 [data-button="play"] {
   position: relative;
-  display: inline-block;
-  min-height: 56px;
-  border-radius: 35px;
+  min-height: 3.5rem;
+  border-radius: 2.188rem;
```

Comment:
Would be great to firm up the answer to find out if we need to round this :) I recommend asking the team.